scripts/render-notes.py

```diff
@@ -77,25 +94,35 @@ def render_notes(
                             break
                         lines.insert(index + 1, rendered)
                         break
-            if not printonly:
+            if not print_only:
                 with open(render_files[render_file]["outfile"], "w") as file:
                     file.writelines(lines)
-        elif render_file == "changelog.txt":
+            if render_file in print_rendered:
+                print(rendered)
+        elif render_file == "changelog" or render_file == "changelog_long":
             stripped = rendered.lstrip()
-            if not printonly:
+            if not print_only:
                 with open(render_files[render_file]["outfile"], "x") as file:
                     file.write(stripped)
-            print(stripped)
+            if render_file in print_rendered:
+                print(stripped)
 
 
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        "--print",
-        "-p",
+        "--print-only",
+        "-po",
         action="store_true",
         help="Only print the processed release notes",
     )
```

Comment:
This was mainly for local debugging, I think one of the arguments is sufficient.